app/src/main/java/com/mobileasone/dagger2workshop/di/RepositoryFactory.kt

```diff
@@ -1,16 +0,0 @@
-package com.mobileasone.dagger2workshop.di
-
-import com.mobileasone.dagger2workshop.data.local.NotesLocalMemoryRepository
-import com.mobileasone.dagger2workshop.domain.repositories.NotesRepository
-
-object RepositoryFactory {
-
-    private val notesLocalMemoryRepositoryInstance: NotesRepository by lazy { initNotesRepository() }
-
-    fun createNotesRepository(): NotesRepository = notesLocalMemoryRepositoryInstance
```

Comment:
Now we can remove RepositoryFactory since we have the [SampleAppComponent](https://github.com/Jamargle/Dagger2Workshop/pull/5/files#diff-f6c7ac2e2280f5df175fb97e75c929e6) that provides the instance of [NotesRepository](https://github.com/Jamargle/Dagger2Workshop/blob/step2-di-with-dagger2/app/src/main/java/com/mobileasone/dagger2workshop/data/local/NotesLocalMemoryRepository.kt) everywhere in the app through dependency injection if we do the setup for it.